downstream/assemblies/platform/assembly-aap-activate.adoc

```diff
@@ -11,8 +11,10 @@ ifdef::context[:parent-context: {context}]
 . Use your Red Hat customer or Satellite credentials when you launch {PlatformNameShort}.
 . Upload a subscriptions manifest file either using the {PlatformName} interface or manually in an Ansible playbook.
 
-include::platform/proc-aap-activate-with-credentials.adoc[leveloffset=+1]
-include::platform/proc-aap-activate-with-manifest.adoc[leveloffset=+1]
+To activate {PlatformnameShort} using credentials, see link:{URLCentralAuth}/assembly-gateway-licensing#proc-aap-activate-with-credentials[Activate with credentials].
```

Comment:
There is a typo in the attribute; it should have an uppercase N for name: {PlatformNameShort}